ServerCore/Pages/Puzzles/Play.cshtml

```diff
@@ -14,28 +14,28 @@
 }
 <style>
     .radioButton {
-        padding: 0px 5px;
-        color: inherit;
-        text-decoration: none;
+    padding: 0px 5px;
+    color: inherit;
+    text-decoration: none;
     }
 
-        .radioButton:hover {
-            color: inherit;
-            text-decoration: none;
-            cursor: pointer;
-        }
+    .radioButton:hover {
+    color: inherit;
+    text-decoration: none;
+    cursor: pointer;
+    }
 </style>
 
 @if (Model.Event.AllowBlazor && canUsePresence && Model.Team != null)
 {
     <script>
         window.showPresence = (allPresence) => {
-            allPresence.forEach(p => {
-                let presenceId = document.querySelector(`#presence-${p.puzzleId}`);
-                if (presenceId) {
-                    presenceId.textContent = p.presenceText;
-                }
-            });
+        allPresence.forEach(p => {
```

Comment:
indenting: ew. I know this is probably some automatic formatting thing and not intentional, but still.

Reply:
Yeah, that's the default for some reason. I put it back because I'm with you, it's a lot harder to parse when it's flat.